src/spec/payid-aliases.md

```diff
@@ -0,0 +1,194 @@
+category: std
+
+pi: [toc, sortrefs, symrefs, comments]
+smart_quotes: off
+
+area: security
+author:
+      
+  -
+    ins: J. Romero
+    name: Javi Romero
+    org: Ripple
+    street: 315 Montgomery Street
+    city: San Francisco
+    region: CA
+    code: 94104
+    country: US
+      
+normative:
+    RFC2119:
+    RFC3629:
+    RFC3986:
+    RFC5234:
+    RFC5890:
+    RFC5892:
+    RFC8264:
+    PAYID-DISCOVERY: 
+      title: "The PayID Discovery Protocol"
+      target: https://tbd.example.com/
+      author:
+        ins: D. Fuelling
+        fullname: David Fuelling
+    PAYID-PROTOCOL: 
+      title: "PayID Protocol"
+      target: https://tbd.example.com/
+      author:
+         - ins: A. Malhotra
+           fullname: Aanchal Malhotra
+         - ins: D. Schwartz
+           fullname: David Schwartz
+    VERIFIABLE-PAYID: 
+       title: "Verifiable PayID Protocol"
+       target: https://tbd.example.com/
+       author:
+         - ins: A. Malhotra
+           fullname: Aanchal Malhotra
+         - ins: D. Schwartz
+           fullname: David Schwartz
+    UNICODE:
+      title: "The Unicode Standard"
+      target: http://www.unicode.org/versions/latest/
+      author:
+        surname: The Unicode Consortium
+        fullname: The Unicode Consortium
+ 
+informative:
+
+--- note_Feedback
+  This specification is a draft proposal and is part of the 
+  [PayID Protocol](https://payid.org/) initiative. Feedback related to this 
+  document should be sent in the form of a Github issue at: 
+  https://github.com/payid-org/rfcs/issues.
+
+--- abstract
+  This specification defines the 'payid' aliases.
+   
+--- middle
+
+# Introduction
+
+   Although the PayID protocol supports returning multiple addresses in the Payment Account Information Response for a given Network and Environment,
+   it might be useful to users to be able to create groups within a PayID that can separate accounts by its purpose or use.
+   
+   Gmail addresses offer a feature called aliasing that allows the same address to receive emails on some variations of the original address.
+   For example, if a user owns `user@gmail.com` emails sent to the following addresses will all resolve to the primary address:
+
+    user+newsletter@gmail.com
+    user+family@gmail.com
+    user+ads@gmail.com
+    ...
+
+   This kind of feature would be useful in the PayID context, allowing the user to create aliases and group accounts depending on its use or purpose.
+ 
+# Terminology
+   The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+   "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and
+   "OPTIONAL" in this document are to be interpreted as described in
+   [RFC2119][].
+ 
+# Definition
+
+   Create the concept of PayID aliases that will allow the user to create PayID aliases linked to the primary one.
+   For example, by owning `alice$example.com` could allow the user to generate multiple aliases as needed:
```

Comment:
For example, by owning `alice$example.com`, the user could be allowed to generate multiple aliases as needed: